lib/ansible/modules/system/freebsd_update.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2014, Brian Coca <rusik@4ege.ru>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: freebsd_update
+short_description: A wrapper to the freebsd-update utility
+description:
+     - Runs freebsd-update utility and updates freebsd system.
+version_added: "2.8"
+options:
+    action:
+        description:
+            - freebsd-update command to execute — fetch, install, fetch_install, rollback, IDS, cron.
+        required: True
+        choices:
+        - fetch
+        - install
+        - fetch_install
+        - rollback
+        - IDS
+        - cron
+    server:
+        description:
+            - Server to download updates from.
+    basedir:
+        description:
+            - path to filesystem to update.
+    workdir:
+        description:
+            - directory to store working files in.
+    conffile:
+        description:
+            - file to read configuration options from
+    force:
+        description:
+            - force freebsd-update run
+        type: bool
+        default: False
+    key:
+        type: str
+        description:
+            - trust an RSA key with SHA256 of KEY
+author:
+- "Ruslan Gustomiasov (@loqutus)"
+- "Maxim Filimonov (@part1zano)"
+'''
+
+EXAMPLES = '''
+# fetch all available binary updates
+- freebsd_update:
+    action: fetch
+
+# Fetch and install all updates:
+- freebsd_update:
+    action: fetch_install
+    force: yes
+    workdir: /tmp/freebsd_update
+'''
+
+RETURN = ''' # '''
+
+import traceback
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils._text import to_native
+
+
+def main():
+    module = AnsibleModule(
+        argument_spec=dict(
+            action=dict(type='str', required=True, choices=['fetch', 'install', 'fetch_install', 'rollback', 'IDS', 'cron']),
+            server=dict(type='str'),
+            basedir=dict(type='str'),
+            workdir=dict(type='str'),
+            conffile=dict(type='str'),
+            force=dict(type='bool', default=False),
+            key=dict(type='str'),
+        ),
+    )
+
+    action = module.params['action']
+    msg = "Unexpected failure!"
+    server = module.params.get('server')
+    basedir = module.params.get('basedir')
+    workdir = module.params.get('workdir')
+    conffile = module.params.get('conffile')
+    force = module.params.get('force')
+    key = module.params.get('key')
+
+    freebsd_update_bin = module.get_bin_path('freebsd-update', True)
+    cmd = [freebsd_update_bin]
```

Comment:
Include `'--not-running-from-cron'` here:

```suggestion
    cmd = [freebsd_update_bin, '--not-running-from-cron']
```